### DISCUSSION TITLE
Programming Language

### DISCUSSION THREAD

**Discussion by PiyushFx** [Cyfrin/foundry-full-course-cu#1158] [Q&A]
Which programming language should I learn for a better opportunity Rust or Golang.

**Comment by Niferu** [comment]
Hi @PiyushFx 

I would say Rust as it is low level programming language, but it is also a lot harder than Go.

**Comment by xyz899** [comment]
Hey ! It's nice to see fellows wanting to learn ! 

Rust would be the better choice if you are planning to work more in system programming rather than web app. Rust is very nice since it uses a borrower to manipulate data instead of garbage collecting.
